metalnx/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM irods/metalnx:2.6.0
+
+RUN apt-get update && \
+    apt-get install -y \
+        netcat \
+        postgresql-client \
+    && \
+    apt-get clean
+
+COPY metalnx_entrypoint.sh /
+RUN chmod +x /metalnx_entrypoint.sh
+CMD ["/metalnx_entrypoint.sh"]
```

Comment:
Consider adding a comment explaining that this overrides the `CMD` specified in the base image (which does not specify an `ENTRYPOINT`).